src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/AdjudicationsMappingResource.kt

```diff
@@ -284,35 +284,19 @@ class AdjudicationsMappingResource(private val mappingService: AdjudicationMappi
   suspend fun getAllMappings() = mappingService.getAllMappings()
 
   @PreAuthorize("hasRole('ROLE_NOMIS_ADJUDICATIONS')")
-  @DeleteMapping("/mapping/adjudications/all/migration-id/{migrationId}")
+  @PostMapping("/mapping/adjudications/delete-mappings")
```

Comment:
This removed 2 endpoints and adds a new one. But the diff looks likes it has changed the first one.

New endpoint requires a body since we delete a list of mappings so this is a POST not a DELETE